### PR TITLE
Fixes files being installed at wrong path

### PR DESCRIPTION
**Pull request by SupremeTechnopriest**
Some of the files (`.gitignore`, `makefile` and `README.md`) were being created at the `GOPATH` instead of the project directory.  It would be overwritten for each project you create.  This PR fixes that and also adapts a consistent style based on what I assumed to be your intentions. 

Also, I found `myrepoURL` vs `myappName` to be a little misleading. I made some adjustments to make it more succinct. I removed the "my" from the names making them true camelcase and `appName` is only used in templates now.  The project path is generated solely from the `repoUrl`.

### CODE REVIEW

**Comment by SupremeTechnopriest** [comment]
I was going to write my own go generator, but you already have `yo go` so lets collaborate in keeping this project maintained 😄

**Comment by SupremeTechnopriest** [comment]
@bench This is ready to be reviewed and merged in!  Thanks!

**Comment by bench** [comment]
Hi @SupremeTechnopriest, thank u for your contribution. i take a look soon :v:

File: app/index.js [hunk 1]
```diff
@@ -1,79 +1,88 @@
...
+const path = require('path');
+const Generator = require('yeoman-generator');
+const mkdir = require('mkdirp');
 
-module.exports = yeoman.Base.extend({
+module.exports = class extends Generator {
 
-  getmyappName: function() {
+  paths() {
+    this.destinationRoot(process.env.GOPATH || './');
```
Comment (bench) [inline app/index.js:10]: Good point

**Comment by bench** [comment]
OK for me, everything works perfectly.

I'm not a JS dev, so your help is really welcome ;)

**Comment by SupremeTechnopriest** [comment]
My pleasure @Bench. I come from JS and am now entering the wonderful world of golang!  Glad I could help!

**Comment by SupremeTechnopriest** [comment]
Hey @Bench, would you mind bumping the `package.json` version and running `npm publish` so I can use the latest?

**Comment by bench** [comment]
Hi @SupremeTechnopriest, sorry for the delay, currently a little too busy. I will publish it soon.

**Comment by bench** [comment]
version 0.1.3 is released

**Comment by SupremeTechnopriest** [comment]
Thanks! 


Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Benjamin Chenebault <notifications@github.com> Date: 8/10/17  4:50 PM  (GMT-05:00) To: Bench/generator-go <generator-go@noreply.github.com> Cc: Randy Lebeau <randylebeau@gmail.com>, Mention <mention@noreply.github.com> Subject: Re: [Bench/generator-go] Fixes files being installed at wrong path
  (#2) 
version 0.1.3 is released

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Bench/generator-go","title":"Bench/generator-go","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/Bench/generator-go"}},"updates":{"snippets":[{"icon":"PERSON","message":"@Bench in #2: version 0.1.3 is released"}],"action":{"name":"View Pull Request","url":"https://github.com/Bench/generator-go/pull/2#issuecomment-321669394"}}}

**Pull request merged**